deltachat-ios/Controller/WelcomeViewController.swift

```diff
@@ -143,6 +155,24 @@ class WelcomeViewController: UIViewController, ProgressAlertHandler {
         alert.addAction(UIAlertAction(title: String.localized("ok"), style: .default))
         present(alert, animated: true)
     }
+
+    @objc private func cancelButtonPressed() {
+        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
+
+        // take a bit care on account removal:
+        // remove only unconfigured and make sure, there is another account
+        // (normally, both checks are not needed, however, some resilience wrt future program-flow-changes seems to be reasonable here)
+        let selectedAccount = dcAccounts.getSelected()
+        if !selectedAccount.isConfigured() {
+            _ = dcAccounts.remove(id: selectedAccount.id)
+            if self.dcAccounts.getAll().isEmpty {
+                _ = self.dcAccounts.add()
+            }
+        }
+
+        // do not just pop view controller as the stack is empty on restarts
```

Comment:
What does this comment mean? 

`appDelegate.reloadDcContext()` is exactly doing that: popping view controllers

Reply:
the comment should underline that a simple `popViewController()`is not sufficient in all cases (but might work in some), and in fact `appDelegate.reloadDcContext()` does not use `popViewController()`.

but i agree, that this comment is obviously not helpful. i will remove it :)